src/test/eo/org/eolang/txt/text-tests.eo

```diff
@@ -168,3 +168,30 @@
         "Привет, 世界"
       8
     $.equal-to "世"
+
+[] > text-replaced
+  assert-that > @
+    replaced.
+      text
+        "abacaba"
+      "aba"
+      "c"
+    $.equal-to "ccc"
+
+[] > text-replaced-2
+  assert-that > @
+    replaced.
+      text
+        "abacaba"
+      "w"
+      "c"
+    $.equal-to "abacaba"
+
+[] > text-replaced-3
+  assert-that > @
+    replaced.
+      text
+        "abacaba"
+      "abacaba"
+      "abacaba"
+    $.equal-to "abacaba"
```

Comment:
@EugeneDar Let's use Russian or Unicode characters here. Or maybe add a new test

Reply:
@Graur I'll add it.